pytestqt/plugin.py

```diff
@@ -522,13 +522,13 @@ def capture_exceptions():
 
     def hook(type_, value, tback):
         result.append((type_, value, tback))
-        sys.__excepthook__(type_, value, tback)
 
+    old_hook = sys.excepthook
     sys.excepthook = hook
     try:
         yield result
     finally:
-        sys.excepthook = sys.__excepthook__
+        sys.excepthook = old_hook
```

Comment:
I think the behavior here is slightly changed - before, if there was a custom `sys.excepthook` at this point, it wasn't restored (and the system-wide one was always restored). Now the behavior is more correct.

Maybe add a test for this?